src/makers/pisces/pisces.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="93" height="94" viewBox="0 0 93 94" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This image looks exactly like bobcat without the text in the middle, is that intended?

Reply:
Sorry I submitted the wrong commit.

Reply:
This is the branch before adjustment, I apologize for my mistake